src/pages/SyntheticsPage/SyntheticsPage.tsx

```diff
@@ -275,7 +275,9 @@ export function SyntheticsPage(p: Props) {
                 isLoading={isPositionsLoading}
                 savedIsPnlInLeverage={savedIsPnlInLeverage}
                 onOrdersClick={() => setListSection(ListSection.Orders)}
-                onSelectPositionClick={(key) => setActivePosition(getByKey(positionsInfoData, key))}
+                onSelectPositionClick={(key, tradeMode) =>
+                  setActivePosition(getByKey(positionsInfoData, key), tradeMode)
+                }
```

Comment:
hm, setActivePosition is not modified?)

Maybe you missed some changes in the commit?

Reply:
Sorry! I missed the file while commit. I have updated it now.